test/verifier/universal-verifier.v3.test.ts

```diff
@@ -101,4 +100,32 @@ describe("Universal Verifier V3 validator", function () {
 
     // TODO make some test with correct UserID but with wrong challenge
   });
+
+  it("Test linked proofs", async () => {
```

Comment:
May be move this test to a separate file?

Reply:
yes, the tests are already separated for the UV itself, there are V2, V3 and events tests. This is just to keep it consistent.